pages/_app.tsx

```diff
@@ -51,37 +61,50 @@ export default function App({ Component, pageProps }: AppProps) {
 	}, [router.isReady]);
 
 	useEffect(() => {
-		if(!global?.window) return;
+		if (!global?.window) return;
 		const networkStr = getNetwork();
 		setNetwork(networkStr);
 	}, []);
 
-	const SplashLoader = () => <div style={{ background:'#F5F5F5', minHeight: '100vh', minWidth: '100vw' }}>
+	useEffect(() => {
+		// @ts-ignore
```

Comment:
I don't think we need @ts-ignore here.
I would discourage the use of @ts-ignore

Reply:
Yes, but i need to declare a global window type then. copuld not find it and it was basically a quick fix for testing.

Reply:
please use `(window as any)` instead